src/main/java/organice/ui/CommandBox.java

```diff
@@ -61,13 +61,6 @@ private void setStyleToIndicateCommandFailure() {
         styleClass.add(ERROR_STYLE_CLASS);
     }
 
-    /**
-     * Sets the focus to the command box so that user can type in the next command.
-     */
-    public void requestFocus() {
-        commandTextField.requestFocus();
-    }
-
```

Comment:
What does this do and why has it been removed?

Reply:
^Same thoughts as @TheRealRavager